test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -527,10 +544,12 @@ def test_successful_update
         :first_name => 'New First', :last_name => 'New Last',
         :month => 10, :year => 2014
       ),
-      :email => "new@example.com"
+      :email => "new@example.com",
+      :phone => "321-654-0987"
```

Comment:
Minor point but this is the same phone as the old one, so it's not technically testing Update.

Reply:
@curiousepic Are you thinking of the new test with phone from billing? That is using the other phone number. Line 279.

Reply:
Nope - this test is updating the info stored with a card. It is first `store`d with this value for `phone` https://github.com/activemerchant/active_merchant/pull/2564/files/3d128699edc6afd49f7961b32baf638a6e6e2a5d#diff-d3e31ce1d7cbbc102d604bd1b6642dc2R526 And then in the `update`, the same value is sent for `phone` here. So it's not testing that it is truly updated because there's no change in the value.

Reply:
👍 Of course!  Thanks, @curiousepic -- updating now.